jaeger_client/thrift.py

```diff
@@ -90,6 +97,18 @@ def make_tag(key, value, max_length):
         )
 
 
+def _make_traceback_tag(key, value, max_length):
```

Comment:
I would prefer `max_length=constants. MAX_TRACEBACK_LENGTH` and then remove the new parameter from all "process" related methods, because process tags should never have a stack trace in them, and it makes that API look weird all the way up.

Reply:
Done. I'm not sure how to test the behavior of traceback tags getting cut off now though.